sdks/v3-sdk/src/utils/swapMath.ts

```diff
@@ -17,7 +16,7 @@ export abstract class SwapMath {
     sqrtRatioTargetX96: JSBI,
     liquidity: JSBI,
     amountRemaining: JSBI,
-    feePips: FeeAmount
+    feePips: JSBI
```

Comment:
`FeeAmount` is an `enum` used in v3, v4 has no bounds on fees (besides an upper limit?), so I'm decoupling the enum from the swap functions.

Reply:
@ewilz Won't this be a breaking change?